Segmentation Sample Models/YNet/README.md

```diff
@@ -0,0 +1,24 @@
+# YNet
+## Dataset 
+The dataset used is available in [this](https://www.bcsc-research.org/data/variables/) post.
+A total of 87 pathologists diagnosed a randomly assigned subset of 60 slides into four diagnostic categories (benign, atypia, ductal carcinomain situ, and invasive cancer),producing an average of 22 diagnostic labels per case.  The average size of these ROIs is10,000×12,000. We use these 200 ROIs for our data set.
```

Comment:
Highlight the categories, that is don't put in parenthesis as it is a required field while uploading. Also, mention the magnification of samples.